doc/user-guide/commands.xml

```diff
@@ -1022,6 +1022,39 @@
 
 	</bitlbee-setting>
 
+	<bitlbee-setting name="notify_handle" type="string" scope="account">
+		<default>empty</default>
+
+		<description>
+			<para>
+				This setting is available for protocols with e-mail notification functionality. If set to empty all e-mail notifications will go to control channel, if set to some string - this will be the name of a contact who will PRIVMSG you on every new notification.
+			</para>
+		</description>
+
+	</bitlbee-setting>
+
+	<bitlbee-setting name="gmail_notifications" type="boolean" scope="account">
```

Comment:
Why add a GMail-specific one?

Something tells me I will find out later on ... Because this seems sub-optimal to me.